setup.py

```diff
@@ -19,6 +19,7 @@
     author_email="crsmithdev@gmail.com",
     license="Apache 2.0",
     packages=["arrow"],
+    package_data={"arrow": ["py.typed"]},
```

Comment:
Might be better to specify this in `MANIFEST.in`: https://github.com/arrow-py/arrow/blob/master/MANIFEST.in

Reply:
Actually looking at PEP 561, this seems like the way to specify it. Disregard.